cartographer/mapping/internal/optimization/optimization_problem_2d.cc

```diff
@@ -175,7 +175,8 @@ OptimizationProblem2D::~OptimizationProblem2D() {}
 
 void OptimizationProblem2D::AddImuData(const int trajectory_id,
                                        const sensor::ImuData& imu_data) {
-  imu_data_.Append(trajectory_id, imu_data);
+  // IMU data is currently not used in 2D optimization, so we ignore this part
```

Comment:
nit: better remove words like "currently" and "for now" as it is implied: predicting the future is futile unless you already have a plan to add it back in which case this plan should be described in a TODO. Also more concise:
```
// IMU data is not used in 2D optimization.
```